composer.json

```diff
@@ -27,7 +27,7 @@
   "require": {
     "phpunit/phpunit": "^8.4",
     "mikey179/vfsstream": "~1.6.8",
-    "typo3fluid/fluid": "^2.5",
+    "typo3fluid/fluid": "^2.5|^3",
```

Comment:
according to the composer documentation it is "||" 

https://getcomposer.org/doc/articles/versions.md#version-range